### DISCUSSION TITLE
Terminology & Specifying Deck

### DISCUSSION THREAD

**Discussion by debanjandhar12** [debanjandhar12/logseq-anki-sync#117] [Documentation]
Documentation has been moved from this location to this page: https://debanjandhar12.github.io/logseq-anki-sync/docs/usage/deck-tags

**Comment by thompson-vii** [comment]
Logseq namespace is also respected by anki 

![image](https://user-images.githubusercontent.com/31672176/208601608-cec46db8-3f50-4fbb-9670-987432391e5e.png)

![image](https://user-images.githubusercontent.com/31672176/208601627-8b6980b9-8d6d-4805-87fb-daf4c983201f.png)

**Comment by thiswillbeyourgithub** [comment]
Edit : the author implemented it, thanks :)!

Is it possible to add a similar way to specify tags? I have tens of thousands of cards and many tags and use omnivore to read stuff. 

Most often I add cards by batch that have the same tags but are not in the same block. That would help a lot if any tag in the hierarchy (current block, any parent block, page block) could be automatically appended to the card.

What do you think?

Related to my previous comment : https://github.com/debanjandhar12/logseq-anki-sync/issues/147#issuecomment-1518644452

**Reply by ghost** [reply]
I don't know if this will fit your usecase, but tags are synced as well to Anki, so what I do is make filtered decks based upon certain tags, maybe that's a good workaround for your situation

**Comment by UndefinedVoidNull** [comment]
Just a reminder
If you are specifying the deck within an **annotation page** of a pdf file,
then you need to write the deck prop **inside the header props which contains the file and file-path props** rather than writing it in a outside node.
